content/tutorials/jwt-authentication.textile

```diff
@@ -273,41 +276,57 @@ The last part is to add the logic into this form. You'll instantiate Ably's Real
-    e.preventDefault();
-    console.log('Fetching JWT token from auth server')
-    const realtime = new Ably.Realtime({ authUrl: '/auth' });
-    realtime.connection.once('connected', function () {
-    console.log('Client connected to Ably using JWT')
-    alert("Client successfully connected to Ably using JWT auth")
-    })
-}
+  e.preventDefault();
+
+  const username = document.querySelector("#username").value;
+  const password = document.querySelector("#password").value;
+
+  if (!`${username}`.trim() || !`${password}`.trim()) {
+    return alert("Error! Require username and password");
+  }
+
+  // Send user details to the auth server
+  const authUrl = `/auth?clientId=${username}:${password}`;
+  const realtime = new Ably.Realtime({ authUrl });
+
+  console.log("Fetching JWT token from auth server", username, password);
+
+  realtime.connection.once("connected", () => {
+    const { tokenDetails } = realtime.auth;
+    console.log("Client connected to Ably using JWT", tokenDetails);
+    alert("Client successfully connected Ably using JWT auth");
+  });
 </script>
 ```
 
 The code begins by instantiating Ably's Realtime client library, then you assign the path of auth server, @/auth@ to the @authUrl@ parameter. The @/auth@ path of on auth server is responsible for signing a JWT using the private API key and then returning back the @tokenId@, the JWT itself.
 
+It is worth noticing that we also sending the user input to the @authUrl@ as a querystring @(/auth?clientId=${username}:${password})@. This is so that our authentication server can validate the user requesting the JWT. In this simple example we will not cover that, and this detail is to demonstrate how you might send user input to your server to control user connections.
+
 With the returned JWT, the client automatically attempts to connect to Ably using it. If the client successfully authenticates with Ably, the code displays an alert to let the user know that the connection was successful.
 
 Complete code for this tutorial can be found in its "GitHub repository":https://github.com/ably/jwt-auth-node-tutorial.
 
 h2(#step-4-checking-the-output). Step 4 - Checking the output
 
-Now that you have all of your files in place, go ahead and test it out. 
+Now that you have all of your files in place, go ahead and test it out.
 
-1. Start your command line/terminal. 
+1. Start your command line/terminal.
 
-2. Change into your project directory. 
+2. Change into your project directory.
 
 3. Start your server by running:
 
 ```[sh]
 node server.js
 ```
 
-4. Navigate your browser to @http://localhost:3000@, since your Express server is listening on port 3000. 
+4. Navigate your browser to @http://localhost:3000@, since your Express server is listening on port 3000.
 
-You will see the login form. Open the browser JavaScript console if you want to see the logged messages. 
+You will see the login form. Open the browser JavaScript console if you want to see the logged messages.
```

Comment:
Suggest making the opening of the browser console a step in its own right, otherwise there's not really any point in having the `console.log()` statements.